.changeset/spotty-peaches-perform.md

```diff
@@ -0,0 +1,6 @@
+---
+"@ima/plugin-testing-integration": minor
```

Comment:
This is actually a major update and we need to mention, that we need the latest version of IMA 18.